spec/integration/manifest.spec.yml

```diff
@@ -101,6 +101,22 @@ spec:
               default: basic
               examples:
               - basic
+            capabilities:
+              description: |-
+                Stack features that are required by the package to work properly.
```

Comment:
@jsoriano The wording implies to me that _all_ the specified capabilities are required to run a package. Is that a correct interpretation? In other words, these capabilities are `AND`ed together to determine eligibility. If that's not the case then I think the wording should be clarified.